e2e/Uno/HelloUnoWorld.Shared/App.xaml.cs

```diff
@@ -0,0 +1,129 @@
+﻿using HelloUnoWorld.Views;
+using HelloUnoWorld.Dialogs;
+using Microsoft.Extensions.Logging;
+using Prism;
+using Prism.DryIoc;
+using Prism.Ioc;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Navigation;
+
+namespace HelloUnoWorld
+{
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : PrismApplication
+    {
+        /// <summary>
+        /// Initializes the singleton application object.  This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
+        {
+            ConfigureFilters(global::Uno.Extensions.LogExtensionPoint.AmbientLoggerFactory);
+
+            this.InitializeComponent();
+            this.Suspending += OnSuspending;
+        }
+
+        public static string ViewAName = "ViewA";
+
+        protected override void OnLaunched(LaunchActivatedEventArgs args)
+        {
+#if __IOS__
+            // requires Xamarin Test Cloud Agent
+            Xamarin.Calabash.Start();
+#endif
```

Comment:
I'm guessing this would probably also be better off in OnInitialized unless there's a reason we should be waiting for OnLaunched to be called by the framework...

Reply:
`PrismApplicationBase.OnInitialized` is called during OnLaunched in this implementation, and Calabash needs to be initialized during this phase.

Reply:
If Calabash needs to be initialized here rather than the AppDelegate or OnInitialized I'll defer to your judgment